samples/net/wifi/test_certs/ca2.pem

```diff
@@ -0,0 +1,29 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Could you place the certificate changes to a separate commit, also document in the readme file how they were created, and also name the files better, perhaps adding `station` string to name or similar.

Reply:
Yes, of course, need to add more detail info for new added files and raise a separate commit. This will be changed after the details of phase2 discussion.
Hi @krish2718 , This phase2 certificate files same as phase1, as it only for test, I just copied it. should we generate a new set of certificate files?

Reply:
> Hi @krish2718 , This phase2 certificate files same as phase1, as it only for test, I just copied it. should we generate a new set of certificate files?

Yes, please generate separate certs.

Reply:
Updated all certs.